features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Resources/InlineEditableField.jsx

```diff
@@ -112,7 +113,7 @@ class InlineEditableField extends React.Component {
                         </Select>
                     )}
                     {this.props.type === 'input' && <input type='text' onChange={this.handleChange('newValue')} value={this.state.newValue} />}
-                    {this.props.type === 'textarea' && <TextField id='standard-textarea' placeholder='Placeholder' multiline className={classes.textArea} margin='normal' onChange={this.handleChange('newValue')} value={this.state.newValue} />}
+                    {this.props.type === 'textarea' && <TextField id='standard-textarea' placeholder={intl.formatMessage({ id: 'Apis.Details.Resources.InlineEditableField.placeholder', defaultMessage: 'Placeholder' })} multiline className={classes.textArea} margin='normal' onChange={this.handleChange('newValue')} value={this.state.newValue} />}
```

Comment:
Exceeds the maximum column limit 120. 
Check the other places as well. This will throw eslint error.

Reply:
done

Reply:
fix in other places as well

Reply:
fixed in most places, but there are lot of eslint issues that should be fixed. should fix them together in another PR.